infrastructure/modules/psql/variables.tf

```
@@ -0,0 +1,315 @@
+variable "psql_SubnetCidr" {
+  type        = string
+  description = "The CIDR range to use for psql subnet"
+}
+
+variable "organization" {
+  type        = string
+  description = "The organization/service owner name"
+}
+
+variable "environment" {
+  type        = string
+  description = "The environment (ATxx/TTx/YTx)"
+}
+
+variable "location" {
+  type        = string
+  default     = "norwayeast"
+  description = "The location where the resources will be placed."
+}
+variable "psql_NetworkName" {
+  type        = string
+  description = "psql network name"
+}
+
+variable "psql_NetworkResourceGroup" {
+  type        = string
+  description = "psql network name"
+}
+
+variable "psql_ResourceGroup" {
+  type        = string
+  description = "psql resource group"
+}
+
+variable "product" {
+  type        = string
+  description = "Product the resource belongs to"
+}
+
+variable "psql_ComputeSize" {
+  type        = string
+  description = "psql compute size"
+}
+
+variable "psql_HighAvailability" {
+  type        = bool
+  description = "Enable psql high availability"
+}
+
+variable "locks_off" {
+  type        = bool
+  description = "If locks should be on or off"
+}
+
+variable "psql_AdminGroups" {
+  description = "A list of Azure AD groups to be used as administrators"
+  type        = list(string)
+}
+
+variable "psql_StorageSize" {
+  description = "PostgreSQL storage size in MB"
+  type        = number
+}
+
+variable "psql_dbname" {
+  description = "The name of the database"
+  type        = string
+}
+
+variable "log_analytics_workspace_name" {
+  description = "Log analytics workspace name"
+  type        = string
+}
+
+variable "log_analytics_workspace_rg" {
+  description = "Log analytics workspace resource group"
+  type        = string
+}
+ 
+variable "psql_StorageAutoGrow" {
+  type        = bool
+  description = "Enable psql storage auto grow"
+}
+
+variable "psql_GeoRedundantBackup" {
+  type        = bool
+  description = "Enable psql georedundant backup"
+}
+
+variable "psql_DatabaseCollation" {
```

Keep to one type of formating, for terraform i prefer snake_case
```suggestion
variable "psql_database_collation" {
```